average.js

```diff
@@ -1,6 +1,76 @@
+/*function findFilterOutliers(numbers) { 
+
+    var values = numbers.concat();
+
+    values.sort( function(a, b) {
+            return a - b;
+         });
+
+
+    var quartile1 = values[Math.floor((values.length / 4))];
+    var quartile3 = values[Math.ceil(values.length * (3 / 4))-1];
+    var interQaurtileRange = quartile3 - quartile1;
+
+    var maxValue = quartile3 + interQaurtileRange*1.5;
+    var minValue = quartile1 - interQaurtileRange*1.5;
+
+    var filteredValues = values.filter(function(x) {
+        return (x <= maxValue) && (x >= minValue);
+    });
+
+
+    return filteredValues;
+}*/
+
+function getStandardDeviation (array) {
+    const n = array.length
+    const mean = array.reduce((a, b) => a + b) / n
+    return [
+        Math.sqrt(array.map(x => Math.pow(x - mean, 2))
+            .reduce((a, b) => a + b) / n),
+        mean
```

Comment:
Function is called `getStandardDeviation` but it gets the mean as well? Why is there a single function to do both?

Reply:
@numocityadmin Sir, Finding mean is a part of finding standard deviation so I included both in a single function. I will change the name of the function appropriately.